src/docfx/build/page/OutputModel.cs

```diff
@@ -4,25 +4,22 @@
 using System;
 using System.Collections.Generic;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using Newtonsoft.Json.Serialization;
 
 namespace Microsoft.Docs.Build
 {
     [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
-    public class PageModel
+    public class OutputModel : InputMetadata
```

Comment:
Not all fields in InputMetadata is in OutputModel, e.g. monikerRange. Seems clearer to separate them.

Reply:
The previous behavior actually is also put all input metadata into output model, this is just another way to do that.

I will leave this problem into my later PRs. @superyyrrzz

Reply:
> Not all fields in InputMetadata is in OutputModel, e.g. monikerRange. Seems clearer to separate them.

That one is currently solved by `public bool ShouldSerializeMonikerRange() => false;`